src/cljstyle/config.clj

```diff
@@ -333,7 +339,8 @@
      :padding-lines 2}
 
     :eof-newline
-    {:enabled? true}
+    {:enabled? true
+     :trailing-blanks? true}
```

Comment:
After further thought, I think it'd actually be better to default this to `false` and accept that it's a breaking change. The out-of-the-box behavior _should_ be to clean up trailing blank lines.